src/templates/viewer-html-template.ts

```diff
@@ -196,8 +196,8 @@ const template = /* html */ `
 
                         // configure camera
                         this.entity.camera.horizontalFov = true;
-                        this.entity.camera.farClip = bbox.halfExtents.length() * 20;
-                        this.entity.camera.nearClip = this.entity.camera.farClip * 0.001;
+                        this.entity.camera.nearClip = 0.01;
+                        this.entity.camera.farClip = 1000;
```

Comment:
Is it worth just setting these on the `pc-camera` then?

Reply:
yes!